src/Menubar.tsx

```diff
@@ -435,24 +467,27 @@ function PictureInner({ config, onConfigUpdate, disabled }: PictureProps) {
 
   const updateConfig = useCallback(onConfigUpdate || ((config: Partial<api.Config>) => {}), [
```

Comment:
```suggestion
  // call to update the configuration if onConfigUpdate is set. otherwise noop
  const updateConfig = useCallback(onConfigUpdate || ((config: Partial<api.Config>) => {}), [
```